io_scene_godot/converters/animation.py

```diff
@@ -141,15 +142,16 @@ class AnimationPlayer(NodeTemplate):
     def __init__(self, name, parent):
         super().__init__(name, "AnimationPlayer", parent)
         # use parent node as the animation root node
-        self['root_node'] = NodePath(self.get_path(), self.parent.get_path())
```

Comment:
I'm curious as to why parent and self.parent are different. From what I can see of NodeTemplate, they should be the same?

Reply:
Ah, it should be the same. While I develop on this PR, I removed it first and then add it back, so caused some inconsistence, I will fix it